### ISSUE TITLE
Applications with Spaces in the Name

### ISSUE THREAD

**Issue by sfright** [sarriaroman/FabricPlugin#64]
Hello with the latest release 1.0.9 running into a problem with the application name. When running within a CLI environment, the post-build-hook runs unsuccessfully as the path to the Crashalytics folder is invalid.

```
#!/bin/sh
My Project Folder/Plugins/cordova-fabric-plugin/Fabric.framework/run ######### #######
```

This results in the output...

My Project Folder/Plugins/cordova-fabric-plugin/Fabric.framework/run: No such file or directory

If i open up the hook script and escape the path the process works fine as well as if i downgrade the plugin to 1.0.8

**Comment by hung-doan** [comment]
@sfright PR is available for this issue https://github.com/hung-doan/FabricPlugin/commit/babd27a13b64bf0fd33f687c1d40259b98d9633d

**Comment by RobertYoung** [comment]
Looks like this is broken again since v1.1.4

**Comment by sarriaroman** [comment]
Solved with version 1.1.6. Thanks for your feedback
